packages/paste-website/src/components/site-wrapper/SiteHeader.tsx

```diff
@@ -29,34 +42,64 @@ export const SiteHeader: React.FC = () => {
   return (
     <Box
       as="aside"
-      backgroundColor="colorBackgroundBody"
-      borderColor="colorBorderLight"
-      borderStyle="none none solid none"
-      borderBottomWidth="borderWidth10"
-      paddingLeft="space200"
-      paddingRight="space80"
-      paddingTop="space50"
-      paddingBottom="space50"
+      display="flex"
+      alignItems="center"
+      backgroundColor="colorBackgroundPrimaryDarkest"
       position="sticky"
-      top={`${getMainContentComputedOffset(isPasteTheme)}px`}
+      top={`${getMainContentComputedOffset()}px`}
       zIndex="zIndex80"
     >
-      <Box display="flex" justifyContent="space-between" alignItems="center" flexDirection="row">
+      <Box display="flex" alignItems="center" paddingRight="space70" paddingLeft="space70" minWidth="sizeSidebar10">
+        <MediaObject verticalAlign="center">
+          <MediaFigure spacing="space40">
+            <PasteIconInverse color={theme.textColors.colorTextInverse} display="block" size={42} />
+          </MediaFigure>
+          <MediaBody>
+            <Text as="h2" fontSize="fontSize40" lineHeight="lineHeight40" color="colorTextInverse">
+              <StyledLink to="/">Paste</StyledLink>
+            </Text>
+            <Text
+              as="h3"
+              fontSize="fontSize20"
+              fontWeight="fontWeightNormal"
+              lineHeight="lineHeight20"
+              color="colorTextInverse"
+            >
+              Design System
+            </Text>
+          </MediaBody>
+        </MediaObject>
+      </Box>
+      <Box
+        display="flex"
+        justifyContent="space-between"
+        alignItems="center"
+        flexDirection="row"
+        borderLeftColor="colorBorderInverseDarker"
+        borderLeftStyle="solid"
+        borderLeftWidth="borderWidth10"
+        paddingTop="space60"
+        paddingRight="space70"
+        paddingBottom="space60"
+        paddingLeft="space70"
+        width="100%"
+      >
         <ThemeSwitcher />
         <Box marginTop="space0">
           <Stack orientation="horizontal" spacing="space60">
             <Box minWidth={['size30', 'size30', 'size30', 'size40']}>
               <FormInput
                 id="search-input"
-                type="search"
+                type="text"
                 placeholder="Search..."
                 value={value}
                 onChange={event => {
                   setValue(event.target.value);
                 }}
+                variant="inverse"
+                insertBefore={<SearchIcon decorative={false} title="search" />}
               />
             </Box>
-            <ContactUsMenu />
             <Anchor
```

Comment:
Can you make this anchor display flex or block? I was expecting to see the focus state around the entire thing. Right now it collapses

![image](https://user-images.githubusercontent.com/368249/95142965-cb40b080-0729-11eb-81b1-fa7e412c3084.png)